### DISCUSSION TITLE
Contributing to Open Source

### DISCUSSION THREAD

**Discussion by JustinBui** [cake-build/cake#3828] [Q&A]
Hello, I am a complete beginner at contributing to open-source projects. I came across this repo and would like to see what I can contribute to. I've noticed an emphasis on a lot of front-end here, which is something I'm pretty familiar with in terms of what's being used in this project. If anyone can reach out to guide me, that'd be nice 😃 👍

**Comment by pascalberger** [comment]
Hi @BuiTheBoi. We really appreciate any contributions. 

There are several projects/repos inside the Cake organization:
- [cake-build/cake](https://github.com/cake-build/cake) is the main repository of the Cake runner and command addins (written in C#)
- [cake-build/website](https://github.com/cake-build/website) is the repo for the cakebuild.net website built using Wyam static site generator
- [cake-build/cake-rider](https://github.com/cake-build/cake-rider) is the JetBrains Rider extension written in Kotlin
- [cake-build/cake-vscode](https://github.com/cake-build/cake-vscode) is the Visual Studio code extension written in TypeScript

In all these repos you'll find different issues which might give you ideas what you can contribute.
Make sure to read the [contribution guidelines](https://cakebuild.net/docs/contributing/contribution-guidelines) before sending a pull request.
